src/Compilers/CSharp/Portable/Operations/CSharpOperationFactory.cs

```diff
@@ -2105,5 +2080,16 @@ private IInstanceReferenceOperation CreateCollectionValuePlaceholderOperation(Bo
             bool isImplicit = placeholder.WasCompilerGenerated;
             return new InstanceReferenceOperation(referenceKind, _semanticModel, syntax, type, constantValue, isImplicit);
         }
+
+        private IVariableDeclarationGroupOperation CreateBoundUsingLocalDeclarationOperation(BoundUsingLocalDeclarations boundUsingLocalDeclarations)
+        {
+            bool isAsync = boundUsingLocalDeclarations.AwaitOpt is object;
+            VariableDeclarationKind declarationKid = isAsync ? VariableDeclarationKind.AsynchronousUsing : VariableDeclarationKind.Using;
+            SyntaxNode declarationSyntax = ((LocalDeclarationStatementSyntax)boundUsingLocalDeclarations.Syntax).Declaration;
+            bool declarationIsImplicit = boundUsingLocalDeclarations.WasCompilerGenerated;
+
+            IVariableDeclarationOperation multiVariableDeclaration = new CSharpLazyVariableDeclarationOperation(this, boundUsingLocalDeclarations, _semanticModel, declarationSyntax, type: null, constantValue: default, declarationIsImplicit);
+            return new VariableDeclarationGroupOperation(ImmutableArray.Create(multiVariableDeclaration), declarationKid, _semanticModel, boundUsingLocalDeclarations.Syntax, type: null, constantValue: default, isImplicit: false);
```

Comment:
>false [](start = 222, length = 5)

It is not clear why we simply assume that the node is not implicit. 
In general, it looks like we can share code with ```CreateBoundMultipleLocalDeclarationsOperation```. That method can be changed to take BoundMultipleLocalDeclarationsBase and figure out declaration kind based on the actual kind of the node, plus accessing BoundUsingLocalDeclarations.AwaitOpt if necessary.